SimG4Core/Application/python/NeutronBGforMuonsXS_Term_cff.py

```diff
@@ -0,0 +1,33 @@
+import FWCore.ParameterSet.Config as cms
+
```

Comment:
Hi @civanch - please add some documentation of whats happening in this customize function and how it differs from the others. 

Second, could it not just rely on NeutronBGforMuonsXS_cff.py for easier future maintenance?